blog/_posts/2024-05-14-core-extension.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Introducing Core Kubernetes Extensions for AKS"
```

Comment:
I get it we need to distinguish between 'core' and the other generic extensions wrt degree of integration with aks. on the naming, wondering if 'core' conveys that? thoughts on 'native' or other words? (I don't have a strong preference here, just brainstorming on name)